packages/web-client/tests/integration/components/card-space/user-page/layout-card-container-test.ts

```diff
@@ -0,0 +1,175 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { click, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+import { CARD_EDIT_MODAL_STATES } from '@cardstack/web-client/components/card-space/user-page/card-edit-modal';
+
+const LAYOUT_CARD_CONTAINER = '[data-test-card-space-layout-card-container]';
+const LAYOUT_CARD_CONTAINER_HEADER =
+  '[data-test-card-space-layout-card-container-header]';
+const LAYOUT_CARD_CONTAINER_EDIT_BUTTON =
+  '[data-test-card-space-layout-card-container-edit-button]';
+const LAYOUT_CARD_CONTAINER_HEADER_EDIT_BUTTON =
+  '[data-test-card-space-layout-card-container-header-edit-button]';
+const LAYOUT_CARD_CONTAINER_EDIT_COMPONENT =
+  '[data-test-card-space-card-edit-modal]';
+const LAYOUT_CARD_CONTAINER_EDIT_COMPONENT_HEADER =
+  '[data-test-card-space-card-edit-modal] [data-test-boxel-header]';
+const LAYOUT_CARD_CONTAINER_EDIT_COMPONENT_TITLE =
+  '[data-test-card-space-card-edit-modal-title]';
+
+module(
+  'Integration | Component | card-space/user-page/layout-card-container',
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    hooks.beforeEach(function () {
+      this.setProperties({
+        editable: false,
```

Comment:
minuscule importance but this is set to `true` in all but one test, could make that the default 🤓